doc/start/os-recommendations.rst

```diff
@@ -117,4 +117,4 @@ Testing
 
 .. _CRUSH Tunables: ../../rados/operations/crush-map#tunables
 
-.. _CephFS best practices: ../../cephfs/best-practices
+.. _CephFS best practices: ../../cephfs/mount-using-kernel-driver
```

Comment:
in addition to changing the link, i think we also need to change the text to "Mount CephFS using Kernel Driver".

Reply:
Sure, makes sense. Done.